types/koa-log/koa-log-tests.ts

```diff
@@ -1,10 +1,8 @@
 import Koa = require('koa');
 import koaLogger = require('koa-log');
 
-koaLogger.morgan.token('request-body', (ctx: Koa.BaseContext) => JSON.stringify(ctx.request.body));
```

Comment:
```
interface ExtendableContext extends BaseContext {
        app: Application;
        request: Request;
```

The `request` property is not defined on `BaseContext` (see above). Because of the index signature, `ctx.request` was `any`, and `ctx.request.body` was okay. Without any body parser middleware, `ctx.request.body` isn't okay either!